packages/web-components/src/slider/slider.styles.ts

```diff
@@ -152,52 +122,55 @@ export const styles = css`
     );
   }
 
-  :host([orientation='vertical'][step]) .track::after {
-    inset: -2px 0;
+  .thumb-container {
+    position: absolute;
+    grid-row: 2 / 2;
+    grid-column: 2 / 2;
+    transform: translateX(-50%);
+    left: var(--slider-thumb);
```

Comment:
Is LTR OK here?

Reply:
Yeah since this is vertical. Also Chris added an RTL story and everything looked good.